packages/key-readmodel-writer/src/keyConsumerServiceV1.ts

```diff
@@ -46,18 +46,9 @@ export async function handleMessageV1(
         "metadata.version": { $lte: message.version },
       });
     })
-    .with({ type: "KeyRelationshipToUserMigrated" }, async (message) => {
-      const kid = message.data.keyId;
-      const userId = message.data.userId;
-      await keys.updateOne(
-        {
-          "data.kid": kid,
-          "metadata.version": { $lte: message.version },
-        },
-        { $set: { "data.userId": userId } },
-        { upsert: true }
-      );
-    })
+    .with({ type: "KeyRelationshipToUserMigrated" }, async () =>
```

Comment:
let's move it [here](https://github.com/pagopa/interop-be-monorepo/pull/741/files/6bb7de5f12c56d8b7c9322d5acf8587529ea656f..3340551c71c20af87d22db924b0ed03dd75da0c4#diff-b8ea84ce5a117031a67dd430ab9f1b762717f96520c85015d615c455d12a4c3fR59) to keep all no-ops together